_build/requirements.txt

```diff
@@ -28,7 +28,6 @@ molecule==4.0.1
 molecule-podman==2.0.3
 mypy-extensions==0.4.3
 packaging==21.3
-paramiko==2.11.0
```

Comment:
@cidrblock This will undo your hack, mainly because you forgot to add it to `requirements.in` and I missed that during review.